mhs-reference-implementation/mhs/routing/tests/test_dictionary_cache.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+import mhs.routing.dictionary_cache as dict_cache
+from utilities.test_utilities import async_test
+import time
+
+
+class TestDictionaryCache(unittest.TestCase):
+
+    @async_test
+    async def test_get_empty_value(self):
+        cache = dict_cache.DictionaryCache()
+        value = await cache.retrieve_mhs_attributes_value("ods", "interaction")
+
+        self.assertIsNone(value)
+
+    @async_test
+    async def test_get_basic_value(self):
+        cache = dict_cache.DictionaryCache()
+        self.assertTrue(not cache.cache)
+
+        await cache.add_cache_value("code", "int", "check123")
+        value = await cache.retrieve_mhs_attributes_value("code", "int")
+
+        self.assertIsNotNone(value)
+        self.assertEqual(value, "check123")
+
+    @async_test
+    async def test_timeout(self):
```

Comment:
Test name implies you're testing when a cache entry expires, when what you're actually testing is when the cache entry hasn't expired.

Reply:
Updated the name appropriately